scripts/constants.sh

```diff
@@ -1,10 +1,10 @@
-#!/usr/bin/env bash
 # Copyright (C) 2023, Ava Labs, Inc. All rights reserved.
 # See the file LICENSE for licensing terms.
 
-
-# Set the CGO flags to use the portable version of BLST
-#
-# We use "export" here instead of just setting a bash variable because we need
-# to pass this flag to all child processes spawned by the shell.
-export CGO_CFLAGS="-O -D__BLST_PORTABLE__"
+function set_cgo_flags() {
```

Comment:
My ideas was to be future-proof. In the case that more functions/logic is added to the script, the approach I've taken won't necessitate the refactoring of existing scripts which don't want the newly added logic.

The other approach is to just create a new file that holds the new logic, but because of the file name(constants.sh) I decided that this would be the way to go.